### ISSUE TITLE
Premake5 alpha 6 problem setting target VS version for C# applications.

### ISSUE THREAD

**Issue by ccptleu** [premake/premake-core#362]
I'm using premake5 alpha 6. In my C# console application, I am unable to set the application name according to the target VS version.  For example, the below will create a console executable as ProjName_.exe where the target visual studio version is missing.

    project(ProjName)	
        location (ProjPath)
        filename(ProjName.."_%{_ACTION or ''}") 	
        kind "ConsoleApp"
        language "C#"
        configuration { "x64", "Debug" }     
           targetsuffix "d_$(PlatformToolset)" 
        ...etc

However, this works correctly if I have a C++ console application. For example, the below will create the executable name as ProjectName_v120.exe.

    project(ProjName)
        filename(ProjName.."_%{_ACTION or ''}") 	
        kind "ConsoleApp"
        language "C++"
        configuration { "x64", "Debug" }       
           targetsuffix "d_$(PlatformToolset)" 
        ...etc

**Comment by tvandijck** [comment]
try using the 'targetname' API.

for Heroes of the Storm we do:
project 'Heroes'
    language 'c++'
    kind 'WindowedApp'

    targetname 'HeroesOfTheStorm'
    targetdir 'Versions/Base00000'

**Comment by ccptleu** [comment]
Thanks for your reply! 

I forgot to add that I had also used targetsuffix (I should edit my code above).  Anyways, I tried setting targetname with setting platform toolset but I can't seem to get it to work for C# projects.  Have you tried it for C# project instead of C++?  

I also tried setting targetsuffix or setting targetname "ProjName_$(PlatformToolset)" without setting targetsuffix but that did not seem to work either.  

project(ProjName)   
	kind "ConsoleApp"
	language "C#"
	
	configuration { "x64", "Debug" }	 
		targetname "ProjName"     
		targetsuffix "d_$(PlatformToolset)"
